core/src/components/loading/test/isOpen/loading.e2e.ts

```diff
@@ -0,0 +1,29 @@
+import { test } from '@utils/test/playwright';
+
+test.describe('loading: isOpen', () => {
+  test.beforeEach(async ({ page, skip }) => {
+    skip.rtl('isOpen does not behave differently in RTL');
```

Comment:
Do we need both modes here? If not, could we skip one of them too?

Reply:
I think we can skip one here as well 👍 I need to remember that testing across modes is only valuable for design differences or if we are explicitly checking for mode and doing something different in the implementation.